.circleci/config.yml

```diff
@@ -1,75 +1,222 @@
 version: 2
 jobs:
-  build:
+  build_library:
     docker:
-      - image: circleci/node:10.13.0-browsers
+      - image: circleci/node:12.9.1-browsers
     steps:
       - checkout
       - restore_cache:
           keys:
-            - v1-dependencies-reactnative-{{ checksum "components/package.json" }}
-            - v1-dependencies-reactnative-
+            - v2-dependencies-react-native-{{ checksum "components/yarn.lock" }}
```

Comment:
Should we not be caching based on package.json in case someone forgets to check in an updated yarn.lock file? Or maybe both?